lib/demos/material/picker_demo.dart

```diff
@@ -44,12 +50,18 @@ class _PickerDemoState extends State<PickerDemo> {
         return DateFormat.yMMMd().format(_fromDate);
       case PickerDemoType.time:
         return _fromTime.format(context);
+      case PickerDemoType.range:
+        return DateFormat.yMMMd().format(_fromRange.start) +
+            ' - ' +
+            DateFormat.yMMMd().format(_fromRange.end);
+        break;
     }
     return '';
   }
 
-  Future<void> _showDatePicker() async {
+  Future<void> _showDatePicker(BuildContext context) async {
     final picked = await showDatePicker(
+      useRootNavigator: false,
```

Comment:
So I tested this out and for some reason it uses the landscape layout inside and looks a bit cramped, so actually let's remove this here.